### ISSUE TITLE
Boolean app parameters do not translate to checkboxes

### ISSUE THREAD

**Issue by chrisgorgo** [OpenNeuroOrg/openneuro#11]
![image](https://user-images.githubusercontent.com/238759/29727865-7eb7ed98-898a-11e7-9aff-e69a2f21bd13.png)

results in

![image](https://user-images.githubusercontent.com/238759/29727843-682a587c-898a-11e7-9bec-b6a131bb6709.png)

instead of a checkbox.

If the checkbox is checked this should result in inclusion of `--ica` flag if it is not checked it should not be included.

**Comment by chrisgorgo** [comment]
https://openneuro.dev.sqm.io/datasets/ds000105/versions/00001?app=MRIQC&version=36&job=068ac72d-0e86-4f06-a9d1-f74b6c0a136c

```mriqc [-h] [--version]
             [--participant_label PARTICIPANT_LABEL [PARTICIPANT_LABEL ...]]
             [--session-id SESSION_ID [SESSION_ID ...]]
             [--run-id RUN_ID [RUN_ID ...]] [--task-id TASK_ID [TASK_ID ...]]
             [-m [{T1w,bold,T2w} [{T1w,bold,T2w} ...]]] [-w WORK_DIR]
             [--report-dir REPORT_DIR] [--verbose-reports] [--write-graph]
             [--dry-run] [--profile] [--use-plugin USE_PLUGIN] [--no-sub]
             [--email EMAIL] [-v] [--webapi-url WEBAPI_URL]
             [--webapi-port WEBAPI_PORT] [--upload-strict] [--n_procs N_PROCS]
             [--mem_gb MEM_GB] [--testing] [-f] [--ica] [--hmc-afni]
             [--hmc-fsl] [--fft-spikes-detector] [--fd_thres FD_THRES]
             [--ants-nthreads ANTS_NTHREADS] [--ants-settings ANTS_SETTINGS]
             [--deoblique] [--despike] [--start-idx START_IDX]
             [--stop-idx STOP_IDX] [--correct-slice-timing]
             bids_dir output_dir {participant,group} [{participant,group} ...]
mriqc: error: unrecognized arguments: true```

It seems your are passing `--ica true` or something like this instead of just `--ica`.

**Comment by nellh** [comment]
@chrisfilo The fix from #63 for how the parameters are formatted for the shell is on dev.
